src/mavlink-router/autolog.cpp

```diff
@@ -73,6 +73,7 @@ void AutoLog::stop()
 bool AutoLog::start()
 {
     _system_id = LOG_ENDPOINT_SYSTEM_ID;
+    _add_sys_comp_id(LOG_ENDPOINT_SYSTEM_ID << 8);
```

Comment:
yes, after it initialize the right Log backend it will forward every write_msg() call to the backend.

```
int AutoLog::write_msg(const struct buffer *buffer)
{
    if (_logger != nullptr) {
        return _logger->write_msg(buffer);
    }
    ...
```

So it need a system id so router can route messages that have log backends as target.